news/2024/2024-05-29-osu-catch-world-cup-2024-round-of-32-recap.md

```diff
@@ -0,0 +1,86 @@
+
+Going further down the list, we see Estonia qualifying 32nd just right above the cut off for the first time in ten years. Iceland, the sole debutant of this year, defied all expectations and ended up 30th on the qualification list and are making their world cup debut in the round of 32.
+
+Let's get into the results and the highlights of last week's matches.
+
+## Round of 32 results
+
+In case you missed any of the matches, or are just curious about general results, you can view all of it here.
+
+|                                 Team A |       |       | Team B                      |
+|---------------------------------------:|:-----:|:-----:|:----------------------------|
+|              **Chile** ::{ flag=CL }:: | **5** |   0   | ::{ flag=NZ }:: New Zealand |
+|                 Sweden ::{ flag=SE }:: |   1   | **5** | ::{ flag=AU }:: Australia   |
+| **Russian Federation** ::{ flag=RU }:: | **5** |   0   | ::{ flag=VN }:: Vietnam     |
+|          **Indonesia** ::{ flag=ID }:: | **5** |   1   | ::{ flag=NO }:: Norway      |
+|              **China** ::{ flag=CN }:: | **5** |   3   | ::{ flag=NL }:: Netherlands |
+|             **France** ::{ flag=FR }:: | **5** |   0   | ::{ flag=TR }:: Turkey      |
+|             **Canada** ::{ flag=CA }:: | **5** |   1   | ::{ flag=LV }:: Latvia      |
+|     **United Kingdom** ::{ flag=GB }:: | **5** |   0   | ::{ flag=MX }:: Mexico      |
+|            **Belgium** ::{ flag=BE }:: | **5** |   2   | ::{ flag=PH }:: Philippines |
+|             **Taiwan** ::{ flag=TW }:: | **5** |   2   | ::{ flag=DE }:: Germany     |
+|              **Japan** ::{ flag=JP }:: | **5** |   0   | ::{ flag=TH }:: Thailand    |
+|             **Poland** ::{ flag=PL }:: | **5** |   0   | ::{ flag=MY }:: Malaysia    |
+|            **Finland** ::{ flag=FI }:: | **5** |   0   | ::{ flag=IS }:: Iceland     |
+|          **Argentina** ::{ flag=AR }:: | **5** |   2   | ::{ flag=DK }:: Denmark     |
+|      **United States** ::{ flag=US }:: | **5** |   0   | ::{ flag=ES }:: Estonia     |
+|               **Peru** ::{ flag=PE }:: | **5** |   0   | ::{ flag=ES }:: Spain       |
+
+## Match highlights
+
+### ::{ flag=SE }:: Sweden vs. ::{ flag=AU }:: Australia
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2154793570&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+Starting off the weekend early on Saturday, Sweden faced off against Australia. Sweden, seeded 14th, has joined the tournaments with high hopes as their roster this year managed to acquire their best qualification result since the format was introduced in 2022. However, their adversary Australia has managed to end up in the top 16 of the three past world cups and therefore isn’t to be underestimated.
```

Comment:
when copying from a google doc, you get these curly apostrophes `’`, but they should be straight single quotes `'`